### PR TITLE
Remove unnecessary zero multiplications

### PR DESCRIPTION
**Pull request by YeldhamDev**
This PR removes some segments of code that were multiplying a zero value, all of them related to scaling.

### CODE REVIEW

**Comment by MewPurPur** [comment]
I agree with this, 0 means that the EDSCALE shouldn't matter.

**Review by KoBeWi — APPROVED** [review]

**Comment by RedworkDE** [comment]
On the other hand having these multiplications there, makes it clear that these value should be scaled if someone ever wants to to increase these.

**Comment by YeldhamDev** [comment]
@RedworkDE Considering that they are surrounded by scaled values, I think that's obvious even without the multiplications. Besides, there are already non-multiplied zero values across the codebase, so having these being the exception is odd.

**Comment by RedworkDE** [comment]
Yes, it is probably obvious if something should be scaled when you have some experience with the ui code.

I was mostly just pointing out, that while this code does not do anything, there can can still be reasons for its existence.

But yes, if there are similar fragments with the scale missing, they should be unified.

File: editor/editor_themes.cpp [hunk 1]
```diff
@@ -1600,7 +1600,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_constant("shadow_offset_x", "RichTextLabel", 1 * EDSCALE);
```
Comment (AThousandShips) [inline editor/editor_themes.cpp:1600]: There are some inconsistencies where some times it is `1 * EDSCALE` and some just `EDSCALE`

**Comment by YuriSizov** [comment]
I don't have a strong opinion. At least in editor classes those are definitely there on purpose so that the code is consistent and future changes are safe. And in editor classes it is mostly used consistently, but I do see that the default theme does not use the scale in these cases consistently.

So eh. I don't think this improves things, but I'm also not going to die on this hill.

**Comment by akien-mga** [comment]
No strong opinion either, but I think it's easier to keep consistency without the multiplication than with it, so this might be the way to go.

**Comment by akien-mga** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.0.3. This makes other cherry-picks cleaner and doesn't have any effect on anything.

**Pull request merged**